src/DynamoCoreWpf/ViewModels/Watch3D/AttachedProperties.cs

```diff
@@ -148,21 +270,40 @@ private static void IsFrozenPropertyChanged(DependencyObject obj, DependencyProp
                 }
                 // TODO DYN-973: Need new mechanism to trigger render update after selected/frozen/isolated properties change
 
-                var dynamoGeom3D = geom as DynamoGeometryModel3D;
-                if (dynamoGeom3D != null)
+                var meshGeom3d = geom as DynamoGeometryModel3D;
+                if (meshGeom3d != null)
                 {
                     if ((bool)e.NewValue)
                     {
-                        dynamoGeom3D.Material = HelixWatch3DViewModel.TransparentMaterial;
+                        meshGeom3d.Material = HelixWatch3DViewModel.FrozenMaterial;
                     }
                     else
                     {
-                        dynamoGeom3D.Material = GetIsolationMode(dynamoGeom3D) ? 
-                            HelixWatch3DViewModel.TransparentMaterial : HelixWatch3DViewModel.WhiteMaterial;
+                        OnPropertySetFalse(meshGeom3d);
                     }
 
-                    dynamoGeom3D.RequiresPerVertexColoration = true;
-                    geom = dynamoGeom3D;
+                    meshGeom3d.CullMode = SharpDX.Direct3D11.CullMode.Back;
```

Comment:
Can you add a comment here?

Reply:
this has to do with: https://jira.autodesk.com/browse/DYN-2439
but actually I found we have the same artifacts on master, and switching to backface culling may not be such a great idea for open surfaces... so I will remove this change.